client/src/App.js

```diff
@@ -1,64 +1,84 @@
 import React, { useEffect, useState } from "react";
-import {
-	BrowserRouter as Router,
-	Switch,
-	Route,
-	Link,
-} from "react-router-dom";
+import { BrowserRouter as Router, Switch, Route, Link } from "react-router-dom";
 import "./App.css";
 import "./grid.css";
 import Students from "./Components/Students.js";
-import { getMessage } from "./service";
 import Mentors from "./Components/Mentors.js";
 import NewQuestion from "./Components/NewQuestion.js";
+import Results from "./Components/Results.js";
 
-import Questions from "../src/mockData/Questions.json";
 export function App() {
-	const [questionData, setQuestionData]=useState({});
-	const [route, setRoute]=useState("quizzes");
-	useEffect(()=>{
-		fetch(`http://localhost:3100/api/${route}`)
+	const [questions, setQuestions] = useState(null);
+	const [quizes, setQuizes] = useState(null);
+	if (questions !== null && quizes !== null) {
+	}
```

Comment:
that's basically preveniting from crashing- a primitive props validation. if there are no props it will prevent a part of code from execution as if it starts with no props - app will crash becaye before fetches and State is set props will be null